h2/src/main/org/h2/api/ErrorCode.java

```diff
@@ -1935,7 +1935,29 @@
      */
     public static final int ROW_NOT_FOUND_IN_PRIMARY_INDEX = 90143;
 
-    // next are 90110, 90122, 90144
+    /**
+     * The error with code <code>90144</code> is thrown when
+     * trying to register custom data type if an object with this name already exists.
+     * Example:
+     *+
+     * CREATE VALUE TYPE FOO FOR "org.acme.Foo";
+     * CREATE VALUE TYPE BAR FOR "org.acme.Bar";
+     * CREATE VALUE TYPE BAR FOR "org.acme.Bar";
+     * </pre>
+     */
+    public static final int CUSTOM_DATA_TYPE_ALREADY_EXISTS_1 = 90144;
```

Comment:
Please provide meaningful examples for types. For example H2 does not have builtin support for complex numbers or matrices.